tests/ogg_support.rs

```diff
@@ -1,10 +1,9 @@
-use libflac_sys as ffi;
-
 #[test]
+#[cfg(feature = "build")]
```

Comment:
I was wondering why this is necessary, but I tried it and could confirm that this doesn't work when `build` is off, `ogg` is off but the external `libFLAC` indeed *does* support OGG.

So I guess this line here is necessary, but is linking to `libogg` necessary in `build.rs`?

Reply:
It's not, I appear to have been confused initially so as to how libogg factored into all this.

Reply:
OK, thanks, now the contradiction is resolved.